engine/client/cl_game.c

```diff
@@ -3945,7 +3945,7 @@ qboolean CL_LoadProgs( const char *name )
 
 	// a1ba: we need to check if client.dll has direct dependency on SDL2
 	// and if so, disable relative mouse mode
-#if XASH_WIN32
+#if XASH_WIN32 && !defined(XASH_64BIT)
```

Comment:
If I'm not wrong, we eliminated empty definitions in favor of 1 or 0.

So you can rewrite this to #if XASH_WIN32 && !XASH_64BIT

Or replace by XASH_WIN64. See public/build.h